### PR TITLE
HARVESTER: add embedded ui of rancher and longhorn ui to support page

### PR DESCRIPTION
**Pull request by DaiYuzeng**
### Occurred changes and/or fixed issues

- https://github.com/harvester/harvester/issues/2765

### Areas or cases that should be tested

1. Go to support page
2. Both of rancher link and longhorn link will not be visible.
3. Go to user preference page
4. Enable DEV mode
5. Back to support page
6. Both of rancher link and longhorn link will be visible.
7. Click links will open a new tab with rancher/longhorn embedded UI.

### Screenshot/Video

image (5)

### CODE REVIEW

File: pkg/harvester/pages/c/_cluster/support/index.vue [hunk 1]
```diff
@@ -95,6 +119,28 @@ export default {
               </button>
             </div>
           </div>
+          <div v-if="dev" class="row">
+            <div class="col span-6 box mb-20 box-primary">
```
Comment (WuJun2016) [inline pkg/harvester/pages/c/_cluster/support/index.vue:123]: delete `mb-20`

File: pkg/harvester/pages/c/_cluster/support/index.vue [hunk 2]
```diff
@@ -95,6 +119,28 @@ export default {
...
+              <h2>
+                <a rel="nofollow noopener noreferrer" target="_blank" :href="rancherLink">{{ t('harvester.support.internal.rancher.title') }} <i class="icon icon-external-link" /></a>
+              </h2>
+              <div>
+                <p class="warning">
+                  <t k="harvester.support.internal.rancher.titleDescription" :raw="true" />
+                </p>
+              </div>
+            </div>
+            <div class="col span-6 box mb-20 box-primary">
```
Comment (WuJun2016) [inline pkg/harvester/pages/c/_cluster/support/index.vue:133]: ditto

File: pkg/harvester/pages/c/_cluster/support/index.vue (hunk 1 above)
Comment (WuJun2016) [reply pkg/harvester/pages/c/_cluster/support/index.vue:123]: https://github.com/harvester/dashboard/pull/519/files#diff-5fbeb59bd128d0cc953065ca7b0a2898c87a29800100ab015b7adcbf2868b797L81    
`<div class="box box-primary" :class="{'mb-20': dev }">`

image

image

**Review by WuJun2016 — CHANGES_REQUESTED** [review]
other LGTM

**Review by WuJun2016 — APPROVED** [review]
LGTM, thanks

**Comment by w13915984028** [comment]
@DaiYuzeng 

We talked in the past, should the embedded `alertmanager`, `prometheus` WebUI links are added here as well?

**Comment by w13915984028** [comment]
@WuJun2016 yes. sorry I did not realize the links are here. thanks.

**Pull request merged**